spec/eeny-meeny/middleware_spec.rb

```diff
@@ -70,22 +70,6 @@ def initialize_app(secure: true, secret: 'test', path: '/', same_site: :strict)
       end
     end
 
-    context 'and the request contains a cookie from an undefined experiment' do
-      let(:request) { Rack::MockRequest.new(subject) }
-
-      before(:example) do
-        @original_request_cookies = 'test=abc;eeny_meeny_undefined_experiment_v1=thevaluedoesntmatter;'
-        @response = request.get('/test',
-                                'CONTENT_TYPE' => 'text/plain',
-                                'HTTP_COOKIE' => @original_request_cookies)
-      end
-
-
-      it "instructs the browser to remove through the 'Set-Cookie' header on the response" do
-        expect(@response['Set-Cookie']).to include('eeny_meeny_undefined_experiment_v1=thevaluedoesntmatter; path=/; max-age=0; expires=Thu, 01 Jan 1970 00:00:00 -0000; SameSite=Strict')
```

Comment:
This test still needs to be here. Running it locally then I can see that the reason why it fails for you after upgrading the dependencies is that the expected string changed a bit. Using the latest version of `rack` then `delete_cookie` string looks like this:
```
eeny_meeny_undefined_experiment_v1=thevaluedoesntmatter; path=/; max-age=0; expires=Thu, 01 Jan 1970 00:00:00 GMT; SameSite=Strict
```